spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/MultipleConfigMapsTests.java

```diff
@@ -85,11 +86,10 @@ private static void createConfigmap(KubernetesClient client, String configMapNam
 				.addToData(data).done();
 	}
 
-	// the last confimap defined in 'multiplecms.yml' has the highest priority, so
```

Comment:
I do not agree with this comment, but most probably at time of the writing it made sense. Overall, I do not think that the order in which you define config maps in bootstrap files should matter. Especially, enforcing this in a unit tests is somehow weird.

Reply:
Not sure I agree, order matters a lot when it comes to property sources, can you explain why you dont think it should matter?